src/main/java/com/huawei/cloud/servicestage/client/AuthClient.java

```diff
@@ -65,6 +65,9 @@ public static Token getAuthToken(String region, String username,
         CloseableHttpClient client = HttpClients.createDefault();
         CloseableHttpResponse response = null;
 
+        // update domain value for non IAM users
+        domain = (domain!=null || domain=="")? domain : username;

```

Comment:
domain != null  includes the case of domain=="", isn't ?  Does it mean  line 69 should be the same as the following line?

   domain = (domain!=null )? domain : username;

Reply:
@paulmchen  it is not the same, but I will update the code to use isEmpty() instead which checks both use cases.